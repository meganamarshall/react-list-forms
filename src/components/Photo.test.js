import React from 'react';
import { shallow } from 'enzyme';
import Photo from './Photo';
import images from '../assets/images.jpeg';

describe('photo component', () => {
  it('renders the photo', () => {
    const wrapper = shallow(<Photo photo={images} />);
    expect(wrapper).toMatchSnapshot();
  });
});
