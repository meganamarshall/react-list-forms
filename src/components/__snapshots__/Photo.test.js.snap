// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`photo component renders the photo 1`] = `
<img
  src="/path/image"
/>
`;
